src/stop

```diff
@@ -38,6 +38,13 @@ start_cmds.each do |key, cmd|
 
 end
 
+ensure_socket 'web' do
+  interface 'eth0'
+  port '22'
```

Comment:
Is this the right port? I imagine we'll need to determine which ports their app is listening on. If it's a simple web app, it will always be 8080. If it's a tcp/udp that might be different. Maybe we don't need to worry about tcp/udp?